msprime/ancestry.py

```diff
@@ -1922,3 +1924,63 @@ def __init__(
         self.end_frequency = end_frequency
         self.s = s
         self.dt = dt
+
+
+@dataclasses.dataclass
+class NeutralFixation(ParametricAncestryModel):
+    """
+    A fixation of a neutral mutation has occured in the history of the sample.
+    This will lead to a weak reduction in polymorphism  near the fixed site.
+
+    The model is one of a structured coalescent where selective backgrounds are
+    defined as in
+    `Braverman et al. (1995) <https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1206652/>`_
+    The implementation details here follow closely those in discoal
```

Comment:
Is the scaling identical to Schrider and Kern?  The sweep model wasn't, so any differences here also need to be documented.